acnportal/acnsim/base.py

```diff
@@ -0,0 +1,649 @@
+"""
+This module contains a base class shared by all ACN-Sim objects.
+"""
+import json
+import operator
+import os
+# noinspection PyProtectedMember
+from pydoc import locate
+import warnings
+import pkg_resources
+import pandas
+from pandas.io.common import stringify_path, get_handle, get_filepath_or_buffer
+import numpy
+
+__NOT_SERIALIZED_FLAG__ = '__NOT_SERIALIZED__'
+
+
+# Uses methodology discussed in https://stackoverflow.com/a/16372436
```

Comment:
This code constructs an object that returns an error for any operations, attribute accesses, or function calls, which is used when an attribute that was not completely serialized is deserialized.